java-collections-conversions-2/src/test/java/com/baeldung/modelmapper/UserMappingTest.java

```diff
@@ -0,0 +1,85 @@
+package com.baeldung.modelmapper;
+
+import org.hamcrest.Matchers;
+import org.junit.Before;
+import org.junit.Test;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.TypeToken;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.hasProperty;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
+
+/**
+ * @sasam0320
+ * @description
+ * This class has test methods of mapping Integer to Character list,
+ * mapping user list to DTO list using MapperUtil generic methods and Converter
+ */
+public class UserMappingTest {
+
+    private ModelMapper mapper;
+    private List<User> users;
+
+    @Before
+    public void init() {
+
+        mapper = new ModelMapper();
+        mapper.addMappings(new UserPropertyMap());
+        users = new ArrayList();
+        users.add(new User("b100", "user1", "user1@baeldung.com", "111-222", "USER"));
+        users.add(new User("b101", "user2", "user2@baeldung.com", "111-333", "USER"));
+        users.add(new User("b102", "user3", "user3@baeldung.com", "111-444", "ADMIN"));
+
+    }
+
+    @Test
+    public void testMapIntegerList() {
```

Comment:
user the baeldung conventions for test names. you can find the documentation in the drive